src/mame/layout/segaai.lay

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="touchpadcursor">
+		<image state="1">
+			<data><![CDATA[
+				<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1" height="1" viewBox="0 0 1 1">
+					<g fill="none" stroke="#2222cc" stroke-width="0.06" stroke-opacity="1">
+						<circle cx="0.5" cy="0.5" r="0.1" />
+						<circle cx="0.5" cy="0.5" r="0.47" />
+						<line x1="0.03" y1="0.5" x2="0.4" y2="0.5" />
+						<line x1="0.6" y1="0.5" x2="0.97" y2="0.5" />
+						<line x1="0.5" y1="0.03" x2="0.5" y2="0.4" />
+						<line x1="0.5" y1="0.6" x2="0.5" y2="0.97" />
+						<line />
+					</g>
+				</svg>
+			]]></data>
+		</image>
+	</element>
+	<element name="touchpad">
+		<rect state="0"><color alpha="0" /></rect>
+	</element>
+
+	<view name="default">
+		<screen index="0">
+			<bounds left="0" top="0" right="4" bottom="3" />
+		</screen>
+		<element id="touchpad" ref="touchpad">
+			<bounds x="0" y="3.1" width="4" height="3" />
+		</element>
+		<element id="touchpadcursor" ref="touchpadcursor">
+			<!-- will be positioned by script -->
+			<bounds x="0" y="3.1" width="0.2" height="0.2" />
+			<color alpha="0.7" />
+		</element>
+	</view>
+
+	<script><![CDATA[
+		file:set_resolve_tags_callback(
+				function ()
+					local touchpad = file.views['default'].items['touchpad']
+
+					-- recompute target touchpad cursor size and area when necessary
+					local curxoffs, curyoffs, curxscale, curyscale, curwidth, curheight
+					file.views['default']:set_recomputed_callback(
+							function ()
+								local lbounds = touchpad.bounds
+								curwidth = lbounds.width / 10
+								curheight = lbounds.height / 14
+								curxoffs = lbounds.x0 - (curwidth * 0.5)
+								curyoffs = lbounds.y0 - (curheight * 0.5)
+								curxscale = lbounds.width / 255
+								curyscale = lbounds.height / 255
+							end)
```

Comment:
The cursor will be an odd shape with this – the division factors of 10 and 14 are to make it circular on the BEENA where the book page area has a 5:7 aspect ratio.